include/morphio/types.h

```diff
@@ -52,4 +60,22 @@ using MorphologyVersion = std::tuple<std::string, uint32_t, uint32_t>;
 template <typename T>
 using range = gsl::span<T>;
 
+
+struct CellFamily {
+    struct NEURON {
+        using Type = SectionType;
+        static constexpr int value = 0;
```

Comment:
Those values must correspond to the values of the cell family in the spec.
https://bbpteam.epfl.ch/documentation/projects/Morphology%20Documentation/latest/h5v1.html

Maybe you can add a comment about it in the code.